llvm/test/CodeGen/AIE/aie2/schedule/mov_scl.mir

```diff
@@ -0,0 +1,169 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+# RUN: llc -mtriple=aie2 -run-pass=postmisched %s %topdown-multi -o - | FileCheck %s
+
+
+# expected to not be in the same bundle
```

Comment:
Those tests should be under `schedule/resource/port_name.mir` to stay consistent. We usually have one `.mir` file per port. Could you split this file into multiple ones, one for each port tested? Then with each subsequent commit, we would extend those files to test more conflicts. Sorry to notice this now :/